advanced/am-pattern-1/templates/am-analytics/worker/wso2am-pattern-3-am-analytics-worker-statefulset.yaml

```diff
@@ -72,7 +72,7 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - nc -z localhost 9444
+                - nc -z localhost 7444
```

Comment:
Shall we rename the YAML file? It's incorrect right `wso2am-pattern-3-am-analytics-worker-statefulset.yaml`.  It should be pattern 1 instead of pattern 3.

Reply:
Also, have we tested whether the issue is no longer occurs for all the patterns?.

Reply:
+1

Reply:
When changing this value in pattern 2, analytics statefulset restarts itself and not getting ready. Looking a way to implement this.